src/components/Attributes/Attributes.tsx

```diff
@@ -148,6 +148,27 @@ function getSingleChoices(
   }));
 }
 
+function getFileChoice(attribute: AttributeInput): FileChoiceType {
+  const attributeValue = attribute.value[0];
+
+  const definedAttributeValue = attribute.data.values.find(
```

Comment:
So, most of this function is repeated in `getMultiDisplayValue`, maybe a function to handle this for both? 

```
const getDefinedAttributeValue = (
  attribute: AttributeInput,
  attributeValue,
  withFile: boolean
) => {
  const definedAttributeValue = attribute.data.values.find(
    definedValue => definedValue.slug === attributeValue
  );

  if (definedAttributeValue) {
    const baseValues = {
      label: definedAttributeValue.name,
      value: definedAttributeValue.slug
    };

    return withFile
      ? { ...baseValues, file: definedAttributeValue.file }
      : baseValues;
  }

  return {
    label: attributeValue,
    value: attributeValue
  };
};